app/controllers/placements/schools_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Placements::SchoolsController < Placements::ApplicationController
   def show
-    @school = current_user.schools.find(params.require(:id)).decorate
+    @school = schools.find(params.require(:id)).decorate
+  end
+
+  private
+
+  def schools
+    current_user.support_user? ? Placements::School.all : current_user.schools
   end
```

Comment:
![very scope](https://github.com/user-attachments/assets/560c341d-7a97-4f20-932b-22d053ccc817)

Reply:
I will address this in https://github.com/DFE-Digital/itt-mentor-services/pull/1101